benchmark/Cargo.toml

```diff
@@ -4,11 +4,18 @@ version = "0.1.0"
 authors = ["Jon Gjengset <jon@thesquareplanet.com>"]
 
 [dependencies]
-evmap = { path = "../" }
+evmap = { path = "../", features = ["hashbrown", "smallvec"] }
```

Comment:
This can go away now I think?

Reply:
Probably, just to lessen the number of lines changed.